src/code/intro.asm

```diff
@@ -681,67 +681,63 @@ IntroStage8Handler::
+TitleAttrMap3::
+    db   $9A, $56, $0F
+    db   $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00
+
+IF LANG_JP ;The JP version has a gradient in the logo
+
+TitleAttrMap4::
+    db   $9A, $76, $0F
+    db   $01, $01, $01, $01, $01, $01, $01, $01, $01, $01, $01, $01, $01, $01, $01, $01
+TitleAttrMap5::
+    db   $9A, $96, $0F
+    db   $02, $02, $02, $02, $02, $02, $02, $02, $02, $02, $02, $02, $02, $02, $02, $02
+TitleAttrMap6::
+    db   $9A, $B6, $0F
+    db   $02, $02, $02, $02, $02, $02, $02, $02, $02, $02, $02, $02, $02, $02, $02, $02
+TitleAttrMap7::
+    db   $9A, $D6, $0F
+    db   $02, $02, $02, $02, $02, $02, $02, $02, $02, $02, $02, $02, $02, $02, $02, $02
+
 ELSE
-TILE_A = $00
-TILE_B = $00
+
+TitleAttrMap4::
+    db   $9A, $76, $0F
+    db   $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00
+TitleAttrMap5::
+    db   $9A, $96, $0F
+    db   $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00
+TitleAttrMap6::
+    db   $9A, $B6, $0F
+    db   $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00
+TitleAttrMap7::
+    db   $9A, $D6, $0F
+    db   $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00, $00
+
 ENDC
 
-Data_001_72DE::
-    db   $9A, $76, $0F                            ; $72DE
-    db   TILE_A, TILE_A, TILE_A, TILE_A           ; $72E1
-    db   TILE_A, TILE_A, TILE_A, TILE_A           ; $72E5
-    db   TILE_A, TILE_A, TILE_A, TILE_A           ; $72E9
-    db   TILE_A, TILE_A, TILE_A, TILE_A           ; $72ED
-
-
-Data_001_72F1::
-    db   $9A, $96, $0F                            ; $72F1
-    db   TILE_B, TILE_B, TILE_B, TILE_B           ; $72F4
-    db   TILE_B, TILE_B, TILE_B, TILE_B           ; $72F8
-    db   TILE_B, TILE_B, TILE_B, TILE_B           ; $72FC
-    db   TILE_B, TILE_B, TILE_B, TILE_B           ; $7300
-
-Data_001_7304::
-    db   $9A, $B6, $0F                            ; $7304
-    db   TILE_B, TILE_B, TILE_B, TILE_B           ; $7307
-    db   TILE_B, TILE_B, TILE_B, TILE_B           ; $730B
-    db   TILE_B, TILE_B, TILE_B, TILE_B           ; $730F
-    db   TILE_B, TILE_B, TILE_B, TILE_B           ; $7313
-
-Data_001_7317::
-    db   $9A, $D6, $0F                            ; $7317
-    db   TILE_B, TILE_B, TILE_B, TILE_B           ; $731A
-    db   TILE_B, TILE_B, TILE_B, TILE_B           ; $731E
-    db   TILE_B, TILE_B, TILE_B, TILE_B           ; $7322
-    db   TILE_B, TILE_B, TILE_B, TILE_B           ; $7326
-
-Data_001_732A::
-    dw   Data_001_72DE                            ; $732A
-    dw   Data_001_72CB                            ; $732C
-    dw   Data_001_72F1                            ; $732E
-    dw   Data_001_72B8                            ; $7330
-    dw   Data_001_7304                            ; $7332
-    dw   Data_001_72A5                            ; $7334
-    dw   Data_001_7317                            ; $7336
+TitleAttrMap::
```

Comment:
From the failing build, you may want to rename this symbol where it is used in the code.